.github/workflows/build_deb.yaml

```diff
@@ -0,0 +1,240 @@
+
+      # Rename the alumet-X.X path with correct version before to enter in it
+      - name: Alumet source code downloading from GitHub repository
+        run: |
+          base=$(find . -maxdepth 1 -type d -name "alumet-X.X")
+          if [ -d "$base" ]; then
+              if [ ! -d "${{ env.ALUMET_DIR }}" ]; then
+                  mv -f "$base" "${{ env.ALUMET_DIR }}"
+              fi
+          else
+              echo "\e[31mERROR\e[0m : No directory '${{ env.ALUMET_DIR }}' found."
+              exit 1
+          fi
+
+      - uses: actions/checkout@v4
+        with:
+          repository: alumet-dev/alumet
+          fetch-tags: 'true'
+          ref: 'main' # TODO CHANGE
+          path: '${{ env.ALUMET_DIR }}/alumet'
+
+      - name: Install Rust and dependencies
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+          echo '. "$HOME/.cargo/env"' >> $HOME/.bashrc
+          . "$HOME/.cargo/env"
+          packages=("curl" "git" "build-essential" "protobuf-compiler" "libssl-dev" "pkg-config")
+          for package in "${packages[@]}"; do
+            if ! dpkg -l | grep -q "^ii  $package "; then
+              echo -e "\e[33mWARNING\e[0m : Packages '$package' required. Installation..."
+              apt-get install -y "$package"
+            fi
+          done
+          rustup default stable
+
+      - name: Compile Alumet
+        run: |
+          . "$HOME/.cargo/env"
+          base=$(pwd)
+          cd ${{ env.ALUMET_DIR }}/alumet/agent/
+          mkdir -p ${base}/bin/
+          CARGO_TARGET_DIR=${base}/bin/ ALUMET_AGENT_RELEASE=true cargo build --release -p alumet-agent --bins --all-features
+          cp ${base}/bin/release/alumet-agent ${base}/${{ env.ALUMET_DIR }}/usr/lib/
+
+      - name: Create config file for Alumet
+        run: |
+          . "$HOME/.cargo/env"
+          mkdir -p ${{ env.ALUMET_DIR }}/etc/alumet/
+          ALUMET_CONFIG=${{ env.ALUMET_DIR }}/etc/alumet/alumet-config.toml ./${{ env.ALUMET_DIR }}/usr/lib/alumet-agent --plugins csv,perf,procfs,socket-control config regen
+
+      - name: Grant rights
+        run: |
+          cd ${{ env.ALUMET_DIR }}
+          chmod 755 usr/lib/alumet-agent
+          chmod 755 usr/bin/alumet-agent
+          chmod 644 etc/alumet/alumet-config.toml
+          chmod 644 usr/lib/systemd/system/alumet.service
+
+      - name: Define package name
+        id: define_package_name
+        run: |
+          echo "PACKAGE_NAME=${{ inputs.name }}-agent_${{ inputs.version }}-${{ inputs.revision }}_${{ inputs.arch }}_${{ env.OS_NAME }}_${{ env.OS_VERSION }}.deb" >> $GITHUB_OUTPUT
+
+      # Package construction with dch and dpkg-buildpackage :
+      #  - 'dch' allow to create the debian/changelog file used in Debian package for some dynamic elements
+      #  - 'dpkg-buildpackage' build the Debian package according the changelog
```

Comment:
From `man dch` and `man dpkg-buildpackage`, that does not seem to be exactly the case.
`dch` generates the changelog, and `dpkg-buildpackage` builds the package from `debian/rules`. The only thing that is used from the changelog is the timestamp, which is used to create env var `SOURCE_DATE_EPOCH`.